package.json

```diff
@@ -24,7 +24,10 @@
     "build": "webpack --config webpack.config.js --mode=development",
     "bundle": "webpack --config webpack.config.js --mode=production",
     "copy": "cp package.json dist/ && cp README.md dist/",
-    "production": "npm run bundle && npm run copy"
+    "production": "npm run bundle && npm run copy",
+    "lic_direct": "npx adsk-npm-license-puller --path . --app-name 'NotificationCenter' --verbose --about-box ./license_output/about-box_direct.html --about-box-type desktop --year 2022 --paos ./license_output/paos_direct.csv",
+    "lic_transitive": "npx adsk-npm-license-puller --path . --app-name 'NotificationCenter' --verbose --about-box ./license_output/about-box_transitive.html --about-box-type desktop --transitive --year 2022 --paos ./license_output/paos_transitive.csv",
+    "generate_license": "npm install @adsk/adsk-npm-license-puller --no-save --force && npm run lic_direct && npm run lic_transitive"
```

Comment:
@mjkkirschner Two questions:

- Seems we should run this everytime a direct prod dependency added
- How to leverage the output htmls, should they be bundled or?

Reply:
Hey @QilongTang 
1. the files in license_output are the results from `generate_license`
2. yes, it would be good to run this when prod dep added, or simply whenever a new release is made.
3. today, I will be manually adding these to dynamo, but I think in the future it would be nice if these were included in the npm package and dynamo about box could reference them from the npm package. (I can file a followup for this after 2.16)

Reply:
3 - continued - I think I did not fully answer your question - no I don't see a good reason to bundle these into the minified .js for notification center, instead the goal is to include this about box content in dynamo's about box window so keeping it separate is useful.

Reply:
Yeah, @mjkkirschner my 2 cents is maybe not bundle into the js but like you said, make the out put stuff part of the npm package. So in Dynamo, it is easy to always get the latest. Just thinking about some scaling problem. I am totally supportive you file a follow up for this though.